crates/bevy_transform/Cargo.toml

```
@@ -8,10 +8,14 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+serialize = ["serde", "bevy_math/serialize"]
+
```

Same as above. Also maybe put it after `[dependencies]` as in other places?
```suggestion
[features]
serialize = ["dep:serde", "bevy_math/serialize"]

```